src/keystats/keystatsContainer.ts

```diff
@@ -0,0 +1,11 @@
+import { connect } from 'react-redux'
+
+import { GlobalState } from '../types';
```

Comment:
I would personally prefer to see absolute imports instead of relative, especially for "global" things like state. E.g. `import { GlobalState } from 'state';` rather than `../types` which doesn't make it obvious where they are without knowing the location of the current file.